spec/abstract/AARCH64/ArchDecode_A.thy

```diff
@@ -156,6 +157,9 @@ definition decode_frame_invocation :: "'z::state_ext arch_decoder" where
      then decode_fr_inv_flush label args cte cap extra_caps
      else throwError IllegalOperation"
 
+definition ppn_from_pptr :: "obj_ref \<Rightarrow> ppn" where
+  "ppn_from_pptr p = ucast (addrFromPPtr p >> pageBits)"
```

Comment:
Hm, it's a trade-off. I was trying not to pollute the flow there too much with details about the `ppn` field. But it would fit there and it's a bit random here in Decode. @Xaphiosis what do you think?

Reply:
RISCV64 has:
```
Arch_Structs_A.thy
61:type_synonym pte_ppn_len = 52 (* machine_word_len - pt_bits *)
62:type_synonym pte_ppn = "pte_ppn_len word"
64:definition ppn_len :: nat where
108:definition addr_from_ppn :: "pte_ppn \<Rightarrow> paddr"
```
so no objections to moving this to Arch_Structs_A

Reply:
It doesn't have the equivalent of `ppn_from_pptr` :-). But I'm convinced. Will move it.